agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -182,7 +185,11 @@ public void trackAsync(TelemetryItem telemetry) {
     // batching, retry, throttling, and writing to disk on failure occur downstream
     // for simplicity not reporting back success/failure from this layer
     // only that it was successfully delivered to the next layer
-    getChannelBatcher().trackAsync(telemetry);
+    if (data instanceof MetricsData) {
```

Comment:
Should it be MetricData instead of MetricsData?

Reply:
are u suggesting renaming it?

Reply:
in the metric data.. base type is always 'MetricData'.  maybe rename it later.  I didn't realized we use MetricsData as the data model.